bioimageio/spec/shared/tracking.py

```diff
@@ -0,0 +1,19 @@
+from bioimageio.spec.shared.common import CONSUMER_ID, CONSUMER_VERSION
+
+
+def report_resource_download(resource_id: str):
+    import requests  # not available in pyodide
+
+    uadata = f'&uadata={{"brands":[{{"brand":"{CONSUMER_ID}","version":"{CONSUMER_VERSION or "unknown"}"}}]}}'
+    url = f"https://bioimage.matomo.cloud/matomo.php?download=https://doi.org/{resource_id}&idsite=1&rec=1&r=646242&h=13&m=35&s=20&url=http://bioimage.io/#/?id={resource_id}{uadata}"
+
+    r = requests.get(url)
+    r.raise_for_status()
+
+
+if __name__ == "__main__":
+    model_doi = "10.5281/zenodo.7614645"
+    report_resource_download(model_doi)
+    print(
+        f"https://bioimage.matomo.cloud/?module=API&method=Actions.getDownload&downloadUrl=https://doi.org/{model_doi}&idSite=1&period=day&date=yesterday&format=JSON&token_auth=anonymous"
+    )
```

Comment:
I could not yet see the tracking working...

Reply:
@oeway any idea what I'm doing wrong?

Reply:
There are two things we need to set here:
 1. we need to configure the `date` properly, it was configured to start from `yesterday` and the period is a `day`, so there won't be any traffic for that model. I would change `period` to `year` and `date` to `2023-03-01` (see here: https://developer.matomo.org/api-reference/Piwik/Period)
 2. the report will only be generated every 15 minutes: https://matomo.org/faq/general/faq_41/ so we won't see the report immediately.
 
 You can see an example here (I triggered the report yesterday so it shows up today): 
 https://bioimage.matomo.cloud/?module=API&method=Actions.getDownload&downloadUrl=https://doi.org/test&idSite=1&idCustomReport=1&period=year&date=2023-03-01&format=JSON&token_auth=anonymous